phpstan.neon.dist

```diff
@@ -0,0 +1,164 @@
+includes:
+    - vendor/mahalux/atk4-hintable/phpstan-ext.neon
+
+parameters:
+    level: 5
+    paths:
+        - ./
+    excludes_analyse:
+        - cache/
+        - build/
+        - vendor/
+
+    # TODO review once we drop PHP 7.x support
+    treatPhpDocTypesAsCertain: false
+
+    ignoreErrors:
+        - '~^Unsafe usage of new static\(\)\.$~'
+
+        # TODO these rules are generated, this ignores should be fixed in the code
+        # level 0
+        -
+          message: "#^Class Atk4\\\\Ui\\\\Persistence\\\\UI not found\\.$#"
```

Comment:
these errors must be definitely fixed

I can then look at the at burn and lowest test.